specs/spec_helpers.py

```diff
@@ -56,17 +56,20 @@ def save(self, dst, buffer_size=16384):
 class TestingRequest(Request): # pylint:disable=too-few-public-methods, too-many-ancestors
     """A testing request to use that will return a
     TestingFileStorage to test file uploading."""
+
     @property
     def files(self):
-        d = MultiDict()
-        if not self.__dict__.get('files'):
+        # Consume steam if it didn't happen yet
+        if 'form' not in self.__dict__:
             self._load_form_data()
-        files = self.__dict__.get('files')
 
-        for key, value in files.items(multi=True):
-            d.add(key, TestingFileStorage(filename=value.filename))
+        file_dictionary = MultiDict()
+        files = self.__dict__.get('files')
+        if files:
+            for key, value in files.items(multi=True):
```

Comment:
Vielleicht ein bisschen sprechendere namen als (key, value) damit man wenn man in die methode schaut eine idee hat, was im file_dictionary am ende drin steht (sind das ein ID und ein filename, oder was anderes).